src/components/Domain/domainTable.tsx

```diff
@@ -244,9 +244,7 @@ const DomainTable: React.FC = () => {
                       <TableCell>
                         <Dialog>
                           <DialogTrigger asChild>
-                            <Button variant="outline">
-                              <Eye className="w-4 h-4" />
-                            </Button>
+                            <ButtonIcon variant="outline"></ButtonIcon>
```

Comment:
Varient is not used from prop in the eye button component, it is hard-coded there.
Either remove the prop OR use it instead of hard coding

Reply:
Ok